### DISCUSSION TITLE
[求助]开加速器也解决不了！

### DISCUSSION THREAD

**Discussion by tianxianuo** [share121/inter-knot#1879] [其他]
![Screenshot_2024-08-13-16-34-56-677_mark via](https://github.com/user-attachments/assets/c3634be4-e0cc-4e5e-b1fc-cd6d1635d2c8)
![Screenshot_2024-08-13-16-51-06-891_mark via](https://github.com/user-attachments/assets/2287b251-77f7-4586-a011-8439e6c2fe8e)
昨天还好好的，今天早上就一直弹“文章加载失败”“获取用户失败”之类的红色感叹号，
我一直开着watt Toolkit，也重新下载过很多次绳网跨域助手，还是解决不了。
今天下午还出现了下载1.10的助手结果需要1.8，用户信息还是认不出来。
这是怎么回事？

**Comment by tianxianuo** [comment]
(｡･ˇ_ˇ･｡:)

**Comment by justafish8787** [comment]
改用x浏览器试试

**Comment by Dave-12138** [comment]
去 [绳网.top](https://www.xn--5o0anh.top/)吧，GitHub pages套了个PWA缓存之后就变成这个上古缓存版本了
